understanding/21/reflow.html

```diff
@@ -35,9 +35,10 @@ <h3>Visibility and availability of content</h3>
     	<section>
 
       <h3>Content exceptions for reflow</h3>
-      <p>Content which requires two-dimensional layout for usage or meaning cannot reflow without losing meaning, and is therefore out of scope of this Success Criterion. For example, graphics and video are by their nature two-dimensional. Cutting up an image and stacking the blocks would render the content unusable so that is out of scope. It is acceptable to provide two-dimensional scrolling for such parts of the content. However, it is possible to have these elements stay within the bounds of viewport at zoom levels to 400% (see advisory techniques).</p>
-      <p>Complex data tables have a two-dimensional relationship between the headings and data cells. This relationship is essential to convey the content. This Success Criterion therefore does not apply to whole data tables, and it is acceptable to provide two-dimensional scrolling for them.  However, cells within data tables should fit within the size requirement unless the cell contains types of content that also requires two-dimensional layout for usage or meaning.</p>
-      <p>Interfaces which provide toolbars to edit content need to show both the content and the toolbar in the viewport. Depending on the number of toolbar buttons, the toolbar may need to scroll in the direction of text (e.g., horizontally in a vertically scrolling page). This Success Criterion therefore does not apply to interfaces which provide toolbars.</p>
+
+      <p>Content which requires two-dimensional layout for usage or meaning cannot reflow without losing meaning, and is therefore excepted from the need to be presented without two-dimensional scrolling. For example, graphics and video are by their nature two-dimensional. Cutting up an image and stacking the blocks would render the content unusable. However, it <em>is</em> possible to have these elements stay within the bounds of viewport even as other content zooms to 400% (see advisory techniques).</p>
```

Comment:
would prefer "loss of meaning" because in many cases content would not lose all its meaning when it reflows.